### DISCUSSION TITLE
Courseplay doubt about the sugarcane harvest trailer transfer of cane

### DISCUSSION THREAD

**Discussion by xseriesx1** [Courseplay/Courseplay_FS22#3055] [Q&A]
Hello community, I’m new here so I apologize for any mistakes made here. My question is about the sugarcane harvest in coursePlay for FS22, if there are possibilities for courseplay to handle it well in the sugarcane transfer, being 2 hitched to the tractor. I remember that in fs17 it was possible to leave them alone harvesting sugarcane, and that he drove to the random cane trailer and unloaded it to the waiting point waiting for the harvester to call him

**Comment by Tensuko** [comment]
Right now, sugar cane is not supported.
If we get this done for FS22 is not clear, but when we can get CP over to the next FS, we will work on that.

**Reply by xseriesx1** [reply]
thanks Tensuko!
